### PR TITLE
CS Fixes

### PR DESCRIPTION
**Pull request by chalasr**

### CODE REVIEW

File: src/DependencyInjection/TacticianExtension.php [hunk 1]
```diff
@@ -13,7 +13,7 @@ class TacticianExtension extends ConfigurableExtension
     /**
      * Configures the passed container according to the merged configuration.
      *
-     * @param array $mergedConfig
+     * @param array            $mergedConfig
```
Comment (tyx) [inline src/DependencyInjection/TacticianExtension.php]: These kind of alignment make diff crazy imo

Comment (rosstuck) [inline src/DependencyInjection/TacticianExtension.php]: I'm not a huge fan either but I checked this against a few other League packages and they all use this type of alignment, so we should probably try to be consistent where we can.

File: src/DependencyInjection/Compiler/CommandHandlerPass.php [hunk 2]
```diff
@@ -34,7 +36,7 @@ public function process(ContainerBuilder $container)
         foreach ($container->findTaggedServiceIds('tactician.handler') as $id => $tags) {
             foreach ($tags as $attributes) {
                 if (!isset($attributes['command'])) {
-                    throw new \Exception('The tactician.handler tag must always have a command attribute');
+                    throw new InvalidArgumentException('The tactician.handler tag must always have a command attribute');
```
Comment (rosstuck) [inline src/DependencyInjection/Compiler/CommandHandlerPass.php]: Good move replacing the exception.

File: tests/testapp/AppKernel.php [hunk 3]
```diff
@@ -11,8 +11,8 @@ class AppKernel extends Kernel
     use MicroKernelTrait;
 
     private $config = [];
-
     private $services = [];
+    private $cacheDir;
```
Comment (rosstuck) [inline tests/testapp/AppKernel.php]: Good catch!

**Pull request merged**